docs/resources/samples/excel-samples.md

```diff
@@ -146,6 +146,68 @@ function main(workbook: ExcelScript.Workbook) {
 }
 ```
 
+### Unhide All Rows and Columns
+
+This script get the worksheet's used range, checks if there are any hidden rows and columns, and unhides them. 
+
+```Typescript
+function main(workbook: ExcelScript.Workbook) {
+    // Get the currently selected sheet.
+    const selectedSheet = workbook.getActiveWorksheet();
+
+    // Get the entire data range.
+    const range = selectedSheet.getUsedRange();
+
+    // If the used range is empty, end the script.
+    if (!range) {
+      console.log(`No data on this sheet.`)
+      return;
+    }
+
+    // Log the address of the used range.
+    console.log(`Used range for the worksheet: ${range.getAddress()}`);
+
+    // If no columns are hidden, log message, else, unhide columns
+    if (range.getColumnHidden() == false) {
+      console.log(`No columns hidden`);
+    } else {
+      range.setColumnHidden(false);
+    }
```

Comment:
Having the "if true" case first is easier to read (as opposed to leading with "if false").
```suggestion
    // Unhide any hidden columns.
    if (range.getColumnHidden()) {
      range.setColumnHidden(false);
    } else {
      console.log(`No hidden columns.`);
    }
```

Reply:
Fair enough. In that case, please change range.getColumnHidden() == false to !range.getColumnHidden()

Reply:
Interestingly enough, when I make that change in my sample script to give it a test, I'm encountering the same issue as above. Can you give this a try on your end and confirm you're seeing the same thing when a column (or row, as below) is hidden?

Reply:
Oh, you're right because it's lumping false and null together. Good catch.